docs/getting-started/recurring-tasks.md

```diff
@@ -88,7 +88,11 @@ You can validate that tasks understands your rule by using the `Tasks: Create or
 
 ---
 
-## Repeating a Task Based on the Original Due Date or the Completion Date
+## Make Overdue Recurring Tasks Skip to Next Future Date: use `when done`
```

Comment:
When I first read this header it didn't quite make sense. 

the "skip to next future date" part is confusing. I did a quick search on our current docs for "future date" and it doesn't seem to be a well-defined concept. When reading through the code for Tasks I did find "future date". So I think this is an implementation details thing?

Reply:
Thank you.

In case GitHub doesn't make it obvious, the previous heading (' Repeating a Task Based on the Original Due Date or the Completion Date') is still there, but I made it an H3. That way, the very many existing links to that section - both here in feature requests, and in Discord, will still work.

The new heading tries to give a clearer (I hoped) explanation of the benefits of the feature...

I'm not sure what the best wording is, but users keep asking for the `when done` feature, and they have not recognised it from the previous headings in this page. So I tried to use words that users were using in their questions...

The most recent such request was this one:
https://github.com/obsidian-tasks-group/obsidian-tasks/issues/1642
> Option to have Recurring Tasks due dates skip to the next available future date

Also:
https://github.com/obsidian-tasks-group/obsidian-tasks/issues/1344
> Recurring tasks should have not set next due date in the past

This search in OMG Discord shows up a few more:
`in: task-management "when done"`

https://discord.com/channels/686053708261228577/965681451297304596/1060515263604342854
> Can a task reaper/reappear weekly no matter if it was completed in the previous week?

Reply:
I can see now why I was confused about this header.

There are three cases (afaik) for how recurring tasks _can_ work:

1. use last due date + period
2. use done date + period
3. use done date + min(period, ?) 

where period is maybe something like every week, every month, etc.

the third case is the "skip to next future date" case.

issue #1642 was resolved for the reporter because they used "every week on Saturday when done". The combination of "when done" plus a specific date makes it such that case 3. is going on.

I was confused about the header because both 2. and 3. exist as usages of `when done`, but the header is only speaking about 3.

as the reporter of #1642 mentioned, what would help in the documentation is to have another example, possibly the exact one they used in the issue, i.e. "every week on Sat when done".

Also, I do think the original header is quite clear. Adding `when done` directly into the header is a good idea and could make people notice this much easier.